src/html/boldOrItalic.js

```diff
@@ -94,8 +93,8 @@ function wrapping_bold_italic(tag, placeholders, chunks){
       chunks.before = chunks.before.replace(/(<[^>]+(?: [^>]*)?>)$/, '</' + tag + '>$1');
     }
   }
-}
 
+}
```

Comment:
The bracket should be shifted to the line above or a line added after closing of bracket and before starting of a new function I think. Rest looks good to me. Great Work!! :smile: